utils/luci-app-ffwizard-berlin/root/www/cgi-bin/has_root-pass

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+echo "Content-type: text/plain"
+echo ""
+USER=root
+# taken from http://serverfault.com/a/240961
+grep ${USER} /etc/shadow | awk -F":" '{ if ($2 == "!" || $2 == "*" || $2 == "") \
```

Comment:
This seems to return "password_is_set:no" in case root is locked, too. This seems like a bad idea to me; if someone disabled root password login, he'll have a reason for this? Check only for "".